platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -259,6 +259,14 @@ class EditorEditText : AppCompatEditText {
 
                 return false
             }
+            android.R.id.undo -> {
+                undo()
+                return false
+            }
+            android.R.id.redo -> {
+                redo()
+                return false
+            }
```

Comment:
Maybe update the comment of this fun?

Also the doc of `TextView.onTextContextMenuItem` says:

> Returns: true if the context menu item action was performed.

So I do not see why all the handled cases here are returning `false` and not `true`. But it looks like parents are also returning `false` so 🤷‍♂️ .